src/constants/numerical.js

```diff
@@ -31,13 +31,17 @@ module.exports = {
      * to remind us that not all years and months
      * have the same length
      */
+    ONEMAXYEAR: 31622400000, // 366 * ONEDAY
     ONEAVGYEAR: 31557600000, // 365.25 days
+    ONEMINYEAR: 31536000000, // 365 * ONEDAY
+    ONEMAXQUARTER: 7948800000, // 92 * ONEDAY
     ONEAVGQUARTER: 7889400000, // 1/4 of ONEAVGYEAR
+    ONEMINQUARTER: 7776000000, // 90 * ONEDAY
```

Comment:
Maybe 89 days, in case someone has a quarter that's Feb/Mar/Apr

Reply:
Good call. Revised in https://github.com/plotly/plotly.js/pull/5089/commits/8928851519e1b74dda977cc574abaa9162820d2c.